packages/template-retail-react-app/app/pages/product-detail/index.jsx

```diff
@@ -218,17 +255,15 @@ const ProductDetail = () => {
                 quantity
             }))
 
-            await addItemToBasketMutation.mutateAsync({
-                parameters: {basketId: basket.basketId},
-                body: productItems
-            })
+            await addItemToNewOrExistingBasket(productItems)
 
             einstein.sendAddToCart(productItems)
 
             // If the items were successfully added, set the return value to be used
             // by the add to cart modal.
             return productSelectionValues
         } catch (error) {
+            console.log('error', error)
```

Comment:
This is already in develop, but I wonder if this should be using the new logger instead of console.log?